lib/go-tc/deliveryservices.go

```diff
@@ -245,7 +245,8 @@ type DeliveryServiceV40 struct {
 
 	// TLSVersions is the list of explicitly supported TLS versions for cache
 	// servers serving the Delivery Service's content.
-	TLSVersions []string `json:"tlsVersions" db:"tls_versions"`
+	TLSVersions           []string               `json:"tlsVersions" db:"tls_versions"`
+	GeoLimitCountriesList *GeoLimitCountriesType `json:"geoLimitCountriesList"`
```

Comment:
I don't think we should add a new property for this, the existing `geoLimitCountries` JSON property can be re-used; when `encoding/json` unmarshals into a structure, it uses the same rules as the Go language to determine which property to unmarshal into.

Example: https://go.dev/play/p/chpKFwvn9X0

Because we rely heavily on nesting structures to avoid duplicating unchanged fields, we have a few duplicated data fields in the API structures, and accessing properties of nested structures instead of just directly (i.e. `fooV40.fooV31.Prop` instead of `fooV40.Prop` even if `Prop` only exists on `fooV40.fooV31`) is not supported by `lib/go-tc`.